lib/rex/proto/ssh/connection.rb

```diff
@@ -309,7 +293,8 @@ def cid=(chan_id)
   def fd_rd
     begin
       channel.io[0]
-    rescue
+    rescue => e
+      elog(e)
```

Comment:
I think if these logs were here, it would've saved me a lot of time git bisecting